codegen/generatePythonSDK.bat

```diff
@@ -0,0 +1,19 @@
+cd .\codegen
+
+set sdkfolder=..\SDKs\Python
+
+copy /y Templates\python\.swagger-codegen-ignore %sdkfolder%\.swagger-codegen-ignore
+if exist "%sdkfolder%\asposeimagingcloud\api\" del /S /Q "%sdkfolder%\asposeimagingcloud\api\" || goto :error
+if exist "%sdkfolder%\asposeimagingcloud\models\" del /S /Q "%sdkfolder%\asposeimagingcloud\models\" || goto :error
+java -jar Tools\swagger-codegen-cli-2.3.1.jar generate -i https://api-qa.aspose.cloud/v3.0/imaging/swagger/sdkspec -l python -t Templates\python -o %sdkfolder% -c config.python.json || goto :error
```

Comment:
@s-zubov Please, make rebase with _development_ branch since it has been modified, and take note on the modifications, especially _swagger-codegen-cli-2.3.1.jar_ replacement with version 2.4.5 which will force modification of Python SDK generation script.

Reply:
Thank you, I will check what's new in the development branch. Please, notice that modifications related to on-premise deployment are out of scope for this PR.

Reply:
@s-zubov Yes, you are absolutely right about the scope.

Reply:
Thank you, I've fixed swagger-codegen version in SDK generation script and updated copyright message in templates.